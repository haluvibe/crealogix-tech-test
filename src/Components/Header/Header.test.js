import React from "react";
import { shallow } from "enzyme";
import Header from "./Header";
import { isTSAnyKeyword, exportAllDeclaration } from "@babel/types";

const mockTitle = "mock title";

describe("Header", () => {
  it("renders without errors when given a title prop", () => {
    const wrapper = shallow(<Header title={mockTitle} />);
    expect(wrapper.find("h1.title").text()).toEqual(mockTitle);
    expect(wrapper).toMatchSnapshot();
  });
});
