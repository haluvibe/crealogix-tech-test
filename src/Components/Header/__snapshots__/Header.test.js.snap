// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders without errors when given a title prop 1`] = `
<h1
  className="title"
>
  mock title
</h1>
`;
